### PR TITLE
[149372] Remove utf-8 encoding specification

### PR DESCRIPTION
**Pull request by ttokit**
https://acsmine.tok.access-company.com/redmine/issues/149372

### CODE REVIEW

File: test/core/gear_log/file_handle_test.exs [hunk 1]
```diff
@@ -46,6 +46,16 @@ defmodule AntikytheraCore.GearLog.FileHandleTest do
     assert_file_content([log_msg])
   end
 
+  test "write/2 should accept malformed (non-UTF8) log message" do
+    log_with_malformed_msg = {@log_time, :info, @context_id, <<222>>}
```
Comment (skirino) [inline test/core/gear_log/file_handle_test.exs]: It'd be nicer if we change `<<222>>` to something like `:crypto.strong_rand_bytes(10)`.

**Comment by skirino** [comment]
One minor comment; other than that LGTM

File: test/core/gear_log/file_handle_test.exs (hunk 1 above)
Comment (ttokit) [reply test/core/gear_log/file_handle_test.exs]: fixed in b26931b

**Comment by ymtszw** [comment]
# okay 🚀

**Pull request merged**